config/schedule.yml

```diff
@@ -146,3 +146,7 @@ capture_query_stats:
   description: "Collects Postgres query stats for PGHero (runs every 5 minutes)"
   cron: "*/5 * * * *"
   class: "CaptureQueryStatsWorker"
+notify_about_published_articles:
+  description: "Sends notifications about the new published articles (runs every minute)"
+  cron: "* * * * *"
+  class: "Articles::PublishWorker"
```

Comment:
Reading note: review Articles::PublishWorker to see how it handles state transitions so the next instance of the worker (one minute later) doesn't pick up a subset of the prior publish worker.